jsonargparse/typehints.py

```diff
@@ -918,7 +918,11 @@ def is_private(class_path):
         return "._" in class_path
 
     def add_subclasses(cl):
-        class_path = get_import_path(cl)
+        try:
+            class_path = get_import_path(cl)
+        except (ImportError, AttributeError) as err:  # Attribute is added in case of dot notation imports
+            warning(f"Hit failing import with following trace: {err}")
```

Comment:
```suggestion
            warning(f"Hit failing import with following error: {err}")
```

`{err}` is not a trace. It is only the message of the exception. The warning would show the line number here, but not a complete trace.

Reply:
so use `traceback` or another tool?
btw, I tried just printing the err and it was quite fine...

Reply:
I don't think there is much benefit of printing the traceback here. Just the error message is good enough.